src/components/molecules/ProfileAdvisors.jsx

```diff
@@ -5,7 +5,7 @@ import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
 import Carousel from 'react-material-ui-carousel'
 
 function Advisors({ advisorProfiles }) {
-  const decks = [[0, 1, 2], [3, 4, 5], [6, 0, 1], [2, 3, 4], [5, 6, 0], [1, 2, 3], [4, 5, 6]]
+  const decks = [[0, 1, 2], [3, 4, 5], [6, 7, 0], [1, 2, 3], [4, 5, 6], [7, 0, 1], [2, 3, 4], [5, 6, 7],]
```

Comment:
@drewjhart Do you know what these decks are? How's the logic is working here? We should probably update this logic, this is error-prune and hard to update by the look of it. I'd probably need a bit context to be able to suggest any alternatives

Reply:
Just taking a stab -- am guessing that this was created last summer to handle the profiles displayed in the advisor carousel. Right you are Mani about this being error-prone; there were actually some wrinkles that Cindy and Jason ironed out earlier to get this to work -- definitely something to circle back on.

Reply:
Sounds good. [Here's](https://github.com/rightoneducation/official-website/issues/290) an issue set up to track this so we can prioritize it.